.changelog/13996.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:improvement
+ui: when determining whether to let a user see Secure Variables, check against all paths in all namespaces.
+```
```

Comment:
Secure Variables hasn't been released yet, so we shouldn't include a changelog entry.

Reply:
remove; thanks!